requirements/source.txt

```diff
@@ -70,7 +70,7 @@ django-allauth==0.22.0
 
 django-appconf==1.0.1
 
-django-celery==3.1.16
+django-celery==3.1.17
```

Comment:
This doesn't appear to be tagged in git so I couldn't check it out in vendor. I'll revert this line since we plan to remove this in the near future.